fe/fe-core/src/main/java/com/starrocks/alter/Alter.java

```diff
@@ -473,6 +495,34 @@ private void processRenameMaterializedView(String oldMvName, String newMvName, D
         LOG.info("rename materialized view[{}] to {}, id: {}", oldMvName, newMvName, materializedView.getId());
     }
 
+    private void processChangeMaterializedViewStatus(MaterializedView materializedView, String status) {
```

Comment:
if it's possible to have other status in the future. do we need to make it extensible ?

Reply:
maybe in the future user can set status to inactive or suspend and so on. I don't known. 
I think better make it extensible.